pom.xml

```diff
@@ -54,7 +48,7 @@
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
     <quarkus.platform.version>3.2.12.Final</quarkus.platform.version>
-    <quarkus-quinoa.version>2.3.6</quarkus-quinoa.version>
+    <quarkus-quinoa.version>2.2.5</quarkus-quinoa.version>
```

Comment:
This has been working, but is actually an unsupported/non-recommended combination of versions.

https://github.com/quarkiverse/quarkus-quinoa?tab=readme-ov-file#versioning

Since we're still using Quarkus 3.2 LTS, I also downgraded the Quinoa version here.